csharp/ql/src/Linq/Helpers.qll

```diff
@@ -36,6 +50,11 @@ predicate missedAllOpportunity(ForeachStmt fes) {
   )
 }
 
+/**
+ * Holds if `ForeachStmt` could be converted to a `.Cast()` call. That is, the
```

Comment:
I now see that this is the usual commenting style, so I'll change these. At the same time, I'm not convinced this is a good approach, because a refactoring can easily change the name of the parameter, and the comments will get out of sync.